site/en/blog/avoiding-oversharing-when-screen-sharing/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Avoiding oversharing when screen sharing'
+description: >
+ The web platform now ships with additional controls for privacy preserving screen sharing.
+authors:
+  - beaufortfrancois
+  - eladalon
+date: 2022-09-29
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/fr1l39yWeyjH8uAZlIbC.jpeg
+alt: three person pointing the silver laptop computer photo
+tags:
+  - chrome-105
+  - chrome-107
+---
+
+Sharing tabs, windows, and screens is already possible on the web platform thanks to [`getDisplayMedia()`]. We’re now shipping the following improvements that guide the user away from accidental oversharing:
+
+- Sharing tabs is now the default option, rather than sharing screens.
+- The `displaySurface` option can indicate that the web app prefers to offer a specific display surface type (tabs, windows, or screens).
+- The `surfaceSwitching` option indicates whether Chrome should allow the user to dynamically switch between shared tabs.
+- The `selfBrowserSurface` option can be used to prevent the user from sharing the current tab. This avoids the "hall of mirrors" effect.
+- The `systemAudio` option ensures Chrome only offers relevant audio-capture to the user.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SRzjjJ9AQVQVyjTCXskh.png", alt="Screenshot of the media picker prompt upon calling getDisplayMedia().", width="800", height="524" %}
+  <figcaption>
+    Chrome’s media picker prompt upon calling getDisplayMedia().
+  </figcaption>
+</figure>
+
+## Reordering the selection offered to the user {: #reordering }
+
+When [`getDisplayMedia()`] is called, users are presented with a media picker which allows them to share whichever display surface they would like any tab, window, or screen. Experience shows that the ordering of the offering affects the users’ selection. We are experimenting in Chrome&nbsp;107 on desktop with reordering the picker in order to nudge users towards tabs, which is the best option in the majority of cases.
```

Comment:
"On desktop" might be redundant, as `getDisplayMedia` is only supported on desktop.